### PR TITLE
resume bank pagination 86 & coffee chat status 87

### PR DESCRIPTION
**Pull request by irfanku**

### CODE REVIEW

File: src/Components/Dashboard/Cards/CoffeeChatSelfCard.js [hunk 1]
```diff
@@ -550,7 +550,7 @@ class CoffeeChatSelfCard extends Component {
                         <h3>
                           {this.props.data.chat_status === "RESERVED"
                             ? "VIEW DETAILS"
-                            : "TO BE RESERVED"}
```
Comment (NabaSadiaSiddiqui) [inline src/Components/Dashboard/Cards/CoffeeChatSelfCard.js:553]: Why did you change this text?

Comment (irfanku) [reply src/Components/Dashboard/Cards/CoffeeChatSelfCard.js:553]: I made changes against https://github.com/maxgala/aspire/issues/87, please let me know if I did something wrong here :(

Comment (NabaSadiaSiddiqui) [reply src/Components/Dashboard/Cards/CoffeeChatSelfCard.js:553]: So we should change "VIEW DETAILS" to "RESERVED". We will show "TO BE RESERVED" if `this.props.data.chat_status === "RESERVED"` evaluates to false

**Review by NabaSadiaSiddiqui — APPROVED** [review]
